test/test_datasets.py

```diff
@@ -2168,5 +2168,23 @@ def inject_fake_data(self, tmpdir, config):
         return num_sequences * (num_examples_per_sequence - 1)
 
 
+class EuroSATTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.EuroSAT
+
+    def inject_fake_data(self, tmpdir, config):
+        tmpdir = pathlib.Path(tmpdir)
+
+        wnid = "AnnualCrop"
+        num_examples = 3
+        datasets_utils.create_image_folder(
```

Comment:
The implementation hard codes a `"2750"` folder. Don't we need that here?

Reply:
good catch (I wasn't sure how to use the inherited test case)